packages/ddex/ingester/e2e_test/e2e_test.go

```diff
@@ -0,0 +1,236 @@
+		t.Fatalf("Failed to decode pending release from Mongo: %v", err)
+	}
+	publishDate := time.Date(2023, time.September, 1, 0, 0, 0, 0, time.UTC)
+	assert.Equal(t, publishDate, pendingRelease.PublishDate, "publish_date doesn't match expected")
+	assert.Equal(t, common.CreateTrackRelease{}, pendingRelease.Track, "Unexpected non-empty track release")
+	assert.Equal(t, deliveryID, pendingRelease.DeliveryID.Hex(), "delivery_id doesn't match expected")
+	assert.Equal(t, []string(nil), pendingRelease.Errors, "Expected there to be no errors")
+	assert.Equal(t, common.CreateAlbumRelease{
+		DDEXReleaseRef: "R0",
+		Metadata: common.CollectionMetadata{
+			PlaylistName:        "Present.",
+			PlaylistOwnerID:     "Bmv3bJ",
+			PlaylistOwnerName:   "Theo Random",
+			ReleaseDate:         publishDate,
+			IsAlbum:             true,
+			IsPrivate:           false,
+			Genre:               common.HipHopRap,
+			CoverArtURL:         fmt.Sprintf("s3://audius-test-indexed/%s/resources/A10301A0005108088N_T-1027024165547_Image.jpg", deliveryID),
+			CoverArtURLHash:     "582fb410615167205e8741580cf77e71",
+			CoverArtURLHashAlgo: "MD5",
+		},
+		Tracks: []common.TrackMetadata{
+			{
+				Title:                "Playing With Fire.",
+				ReleaseDate:          time.Time{},
+				Genre:                common.HipHopRap,
+				Duration:             279,
+				Artists:              []common.Artist{{Name: "Theo Random", Roles: []string{"AssociatedPerformer", "MainArtist"}}},
+				ArtistID:             "",
+				ArtistName:           "",
+				ISRC:                 stringPtr("ZAA012300131"),
+				PreviewStartSeconds:  intPtr(48),
+				PreviewAudioFileURL:  fmt.Sprintf("s3://audius-test-indexed/%s/", deliveryID),
+				AudioFileURL:         fmt.Sprintf("s3://audius-test-indexed/%s/resources/A10301A0005108088N_T-1096524256352_SoundRecording_001-001.m4a", deliveryID),
+				AudioFileURLHash:     "8bb2ce119257314a8fcb215a49f14b33",
+				AudioFileURLHashAlgo: "MD5",
+			},
+			{
+				Title:                "No Comment.",
+				ReleaseDate:          time.Time{},
+				Genre:                common.HipHopRap,
+				Duration:             142,
+				ArtistID:             "",
+				ArtistName:           "",
+				Artists:              []common.Artist{{Name: "Theo Random", Roles: []string{"AssociatedPerformer", "MainArtist"}}, {Name: "Thato Saul", Roles: []string{"AssociatedPerformer", "MainArtist"}}},
+				ISRC:                 stringPtr("ZAA012300128"),
+				PreviewStartSeconds:  intPtr(48),
+				PreviewAudioFileURL:  fmt.Sprintf("s3://audius-test-indexed/%s/", deliveryID),
+				AudioFileURL:         fmt.Sprintf("s3://audius-test-indexed/%s/resources/A10301A0005108088N_T-1096524142976_SoundRecording_001-002.m4a", deliveryID),
+				AudioFileURLHash:     "1e9183898a4f6b45f895e45cd18ba162",
+				AudioFileURLHashAlgo: "MD5",
+			},
+		},
+	}, pendingRelease.Album, "Album release doesn't match expected")
+
+	// TODO: Leaving the publisher untested for now
```

Comment:
my thought for testing this would be a "dry run" where the publisher outputs the data it would've uploaded to audius without actually uploading it. open to suggestions

Reply:
sgtm